types/jest/index.d.ts

```diff
@@ -466,3 +702,369 @@ declare namespace jasmine {
         [n: number]: T;
     }
 }
+
+declare namespace jest {
+    // types for implementing custom interfaces, from https://github.com/facebook/jest/tree/dd6c5c4/types
+
+    // https://facebook.github.io/jest/docs/en/configuration.html#transform-object-string-string
+    // const transformer: Transformer;
```

Comment:
Are these commented out declarations helpful? Seems like something that would belong in the test file